src/content/app/tools/blast/BlastPage.tsx

```diff
@@ -38,8 +42,20 @@ The emphasis of this tool is to find regions of sequence similarity, which will
 `;
 
 const BrowserPage = () => {
+  const lastVisitedBlastView = useAppSelector(getLastVisitedBlastView);
+  const dispatch = useAppDispatch();
+  const navigate = useNavigate();
+  const location = useLocation();
   const hasMounted = useHasMounted();
 
+  useEffect(() => {
+    navigate(lastVisitedBlastView);
+  }, []);
```

Comment:
Unless this is a temporary change that you wanted to update in a subsequent PR, we should really be setting the path to the relevant BLAST view in the launchbar button. That's what I meant by saying that the `app` property of the launchbar button isn't adequate to the job anymore.

Reply:
I've removed the redux code for this and moved the concern of path change to `BlastLaunchbarButton` ✅